src/Common/Constants.ts

```diff
@@ -377,6 +378,7 @@ export class Urls {
   public static autoscaleMigration = "https://aka.ms/cosmos-autoscale-migration";
   public static freeTierInformation = "https://aka.ms/cosmos-free-tier";
   public static cosmosPricing = "https://aka.ms/azure-cosmos-db-pricing";
+  public static serverlessInformation = "https://aka.ms";

```

Comment:
TODO: Update url

Reply:
Removing in next iteration since we don't need to show info banner for serverless accounts.